ios/Podfile.lock

```diff
@@ -570,6 +570,8 @@ PODS:
     - React-Core
   - react-native-netinfo (9.4.1):
     - React-Core
+  - react-native-network-info (5.2.1):
```

Comment:
rollback pls

Reply:
Why? It was missing previously when @devkudasov added it for ip address

Reply:
You can see that https://github.com/haqq-network/haqq-wallet/pull/1481 doesn't have updated `Podfile.lock`